server/verify.go

```diff
@@ -126,7 +124,7 @@ func VerifyAttestation(attestation *pb.Attestation, opts VerifyOpts) (*pb.Machin
 	return nil, fmt.Errorf("attestation does not contain a supported quote")
 }
 
-func pubKeysEqual(k1 crypto.PublicKey, k2 crypto.PublicKey) bool {
```

Comment:
why change to `interface{}`? `crypto.PublicKey` is also an empty interface but it at least conveys that the input should be a PublicKey more than `interface{}`

Reply:
Whoops, this was because [`x509.Certificate.PublicKey`](https://pkg.go.dev/crypto/x509#Certificate) is an `interface{}` and that's what we pass to `pubKeysEqual`. I din't realize that `crypto.PublicKey` is also just an empty interface.